docs/openapi-docs/src/test/scala/tapir/docs/openapi/VerifyYamlTest.scala

```diff
@@ -30,15 +30,34 @@ class VerifyYamlTest extends FunSuite with Matchers {
     .out(jsonBody[F1])
 
   test("should match the expected yaml when schema is recursive") {
-    val expectedYaml = noIndentation(Source.fromResource("expected_recursive.yml").getLines().mkString("\n"))
+    val expectedYaml = loadYaml("expected_recursive.yml")
 
     val actualYaml = endpoint_wit_recursive_structure.toOpenAPI("Fruits", "1.0").toYaml
     val actualYamlNoIndent = noIndentation(actualYaml)
 
     actualYamlNoIndent shouldBe expectedYaml
   }
 
-  case class F1(data: List[F1])
+  test("should use custom operationId generator") {
+    def customOperationIdGenerator(pc: Vector[String], m: Method) = pc.map(_.toUpperCase).mkString("", "+", "-") + m.m.toUpperCase
+    val options = OpenApiDocsOptions.Default.copy(customOperationIdGenerator)
+    val expectedYaml = loadYaml("expected_custom_operation_id.yml")
+
+    val actualYaml = RichOpenAPI(
```

Comment:
why are all implicits explicit?

Reply:
Was checking something using `desugar` feature and forget to roll it back. Thanks for the caution.